projects/core/src/occ/adapters/user/occ-user-order.adapter.ts

```diff
@@ -120,4 +122,17 @@ export class OccUserOrderAdapter implements UserOrderAdapter {
       .get<Occ.OrderHistoryList>(url, { params: params })
       .pipe(this.converter.pipeable(ORDER_HISTORY_NORMALIZER));
   }
+
+  public getConsignmentTracking(
+    orderCode: string,
+    consignmentCode: string
+  ): Observable<ConsignmentTracking> {
+    const url = this.occEndpoints.getUrl('consignmentTracking', {
+      orderCode,
+      consignmentCode,
+    });
+    return this.http
+      .get<ConsignmentTracking>(url)
+      .pipe(catchError((error: any) => throwError(error.json())));
```

Comment:
We probably not need that  `catchError((error: any) => throwError(error.json())`, because it won't ever work correctly.

However, we should consider passing it through converter instead for the sake of customizability, i.e. 
`.pipe(this.converter.pipeable(CONSIGNMENT_TRACKING_NORMALIZER));`, where 
`CONSIGNMENT_TRACKING_NORMALIZER` will be just a token in `order/converters.ts`:

```
export const CONSIGNMENT_TRACKING_NORMALIZER = new InjectionToken<
  Converter<any, ConsignmentTracking>
>('ConsignmentTrackingNormalizer');
```